lily4178993/maths-magicians
# Implement Quote Display Component

# Implement Quote Display Component

## Description
This pull request introduces a new feature for the project, which includes the implementation of a Quote Display component. The following tasks have been completed:

- Utilized the [API Ninjas](https://api-ninjas.com/api/quotes) external API to fetch quotes.
- Created a new React component named `Quotes` to handle the display of quotes.
- Implemented loading and error states to enhance user experience.
- Imported the `Quotes` component into the main component alongside the Calculator.

## Checklist
- [x] Utilized the [API Ninjas](https://api-ninjas.com/api/quotes) external API.
- [x] Created the `Quotes` component.
- [x] Implemented loading and error states.
- [x] Imported the `Quotes` component into the main application.
- [x] Ensured the quote is on the same page as the Calculator.
## Screenshot

![Web capture_12-9-2023_184646_localhost](https://github.com/lily4178993/maths-magicians/assets/101261047/cb62d948-f72f-4918-9629-95e22be80f82)

## Reviewer Notes
Please review the implementation of the Quotes component to ensure it correctly fetches and displays quotes from the API. Verify that loading and error states are functioning as expected.

## Additional Information for Reviewers

This project relies on environment variables for sensitive information, such as API keys. These variables are not included in this pull request for security reasons. Please refer to the project's documentation for details on setting up the necessary environment variables.

If you are a reviewer, please ensure that you have access to the required environment variables to fully test and review this pull request.


## How to Test
To test the Quotes component:

1. Ensure you have cloned this branch and installed dependencies.
2. Run the development server:
   ```bash
   npm start
   ```
3. Open your web browser and access the application.
